warehouse/templates/manage/manage_project_base.html

```diff
@@ -46,17 +46,16 @@ <h3 class="sidebar-section__title">
           <div class="package-snippet package-snippet--margin-bottom">
             <div class="split-layout split-layout--middle split-layout--spaced">
               <div>
-                <h1 class="package-snippet__title package-snippet__title--page-title">{{ project.name }}</h1>
                 {% set release = project.releases[0] if project.releases else None %}
                 {% if release %}
+                  <h1 class="package-snippet__title package-snippet__title--page-title">
```

Comment:
@berkerpeksag The issue is that this line and the following lines are indented 2 spaces too many. All the jinja tags are removed, so this ends up being something like:

```html
<div class="package-snippet package-snippet--margin-bottom">
  <div class="split-layout split-layout--middle split-layout--spaced">
    <div>
        <h1 class="package-snippet__title package-snippet__title--page-title">
```

Reply:
Oh, right, I completely missed that. I shouldn't write code when I'm sick :) Thanks!